### ISSUE TITLE
apache2_module module reenabled already enabled module

### ISSUE THREAD

**Issue by ryan-lane** [ansible/ansible#7857]
##### Issue Type:

Bug Report
##### Ansible Version:

ansible 1.6.3
##### Environment:

Ubuntu 14.04
##### Summary:

When using apache2_module for mpm_prefork a change occurs every playbook run, even though the module is already enabled.
##### Steps To Reproduce:

``` yaml
- name: Ensure mpm_prefork is enabled
  apache2_module: name=mpm_prefork
```
##### Expected Results:

No change during playbook run.
##### Actual Results:

A change occurs every run, even though mpm_prefork is already enabled:

```
# a2enmod mpm_prefork
Considering conflict mpm_event for mpm_prefork:
Considering conflict mpm_worker for mpm_prefork:
Considering conflict mpm_itk for mpm_prefork:
Module mpm_prefork already enabled
```

**Comment by jimi-c** [comment]
I believe this is a duplicate of #7736?

**Comment by ryan-lane** [comment]
Possibly, though for me only the mpm_prefork module is having issues.

**Comment by jimi-c** [comment]
# Closing This Ticket

Hi!

We believe the commit merged in via PR #7923 should resolve this problem for you. This will be included in the next major release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
